### ISSUE TITLE
Extend allocation labels to include kubernetes recommended labels 

### ISSUE THREAD

**Issue by nikovacevic** [kubecost/cost-analyzer-helm-chart#698]
**What problem are you trying to solve?**
Kubecost could make it easier to aggregate by common k8s labels; e.g.
```yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  labels:
    app.kubernetes.io/name: mysql
    app.kubernetes.io/instance: mysql-abcxzy
    app.kubernetes.io/version: "5.7.21"
    app.kubernetes.io/component: database
    app.kubernetes.io/part-of: wordpress
    app.kubernetes.io/managed-by: helm
```

**Describe the solution you'd like**
Include the [k8s recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) in the list of configurable labels, which then appear in the UI as aggregation options (as Name, Instance, Component, etc.).

**Describe alternatives you've considered**
Users can manually use the API to do this today, so there is that workaround.

**How would users interact with this feature?**
The options would appear by default in allocation aggregation options. They could then customize the underlying labels to meet their needs, if necessary.

**Comment by kirbsauce** [comment]
@nikovacevic , is this somewhat addressed with the label functionality in the new multi-aggregation?

**Comment by nikovacevic** [comment]
Multi-agg UI will certainly let you use those labels to aggregate, so in a sense it does. This was reported by a user, though, and I forget the context, so it may be that they really did want more configurable label aliases, like "instance" and "version". Wish I could remember, exactly.

**Comment by nikovacevic** [comment]
Given the precise wording here, I think we could consider this closed by mutli-agg UI.

**Comment by kirbsauce** [comment]
thanks @nikovacevic , closing!
